mailpile/crypto/aes_utils.py

```diff
@@ -99,7 +99,10 @@ def aes_ctr_decryptor(key):
 try:
     aes_ctr_encryptor, aes_ctr_decryptor = make_cryptography_utils()
 except ImportError:
-    aes_ctr_encryptor, aes_ctr_decryptor = make_pycrypto_utils()
+    try:
+        aes_ctr_encryptor, aes_ctr_decryptor = make_pycrypto_utils()
+    except ImportError:
+        raise ImportError("Please pip install cryptography (or pycrypto)")
```

Comment:
I like short messages, they're easier to parse since they fit on one line. ;-)  This is only affects the tiny group of users who installed Mailpile >2 months ago and are upgrading after this gets merged, so not worth spending too much time on. All recent/new installs should pick up cryptography from requirements.txt... so not worth polishing too much.